visualization.shadertoy/resources/language/resource.language.en_gb/strings.po

```diff
@@ -9,117 +9,228 @@ msgstr ""
+#: visualization.shadertoy/resources/settings.xml
+msgctxt "#30038"
+msgid "[B][I]JSON file for user defined shader lists[/I][/B][CR][CR]The file [B]presets.json[/B] is used to define custom shaders.[CR]A custom path to the shader files (GLSL) can be declared in the add-on settings.[CR][CR]Contents of [B]presets.json[/B] should be:[CR][I]  1.[/I] Name of preset. Either hardcoded string or defined in [I]resources/language/.../strings.po[/I][CR][I]  2.[/I] GLSL shader to use. if no custom path was declared, the default [B]resources/[/B] will be used. Set custom path to './' to use shaders from the same path as your [B]presets.json[/B] file.[CR][I]  3.[/I] Channel value 0. Defines the texture to use (e.g. PNG file). Can also be set to \"audio\" to pass related stream data within texture. The PNG and GLSL files should be located in the same directory.[CR][I]  4.[/I] Channel value 1 definition.[CR][I]  5.[/I] Channel value 2 definition.[CR][I]  6.[/I] Channel value 3 definition.[CR][I]  7.[/I] [I][Optional][/I] append string [B]\"gl_only\"[/B] to make this preset only available to OpenGL.[CR][CR][B]Example:[/B][CR][COLOR white][COLOR green]{[/COLOR] [COLOR blue]\"presets\"[/COLOR]:[[CR][COLOR gray]/* Default way used by the addon itself */[/COLOR][CR][ [COLOR red]\"The Disco Tunnel by poljere\"[/COLOR],  [COLOR red]\"discotunnel.frag.glsl\"[/COLOR], [COLOR red]\"tex02.png\"[/COLOR], [COLOR red]\"tex15.png\"[/COLOR], [COLOR red]\"audio\"[/COLOR], [COLOR red]\"\"[/COLOR] ],[CR][COLOR gray]/* Description defined in strings.po */[/COLOR][CR][ 30100, [COLOR red]\"audioeclipse.frag.glsl\"[/COLOR], [COLOR red]\"audio\"[/COLOR], [COLOR red]\"\"[/COLOR], [COLOR red]\"\"[/COLOR], [COLOR red]\"\"[/COLOR] ],[CR][COLOR gray]/* Define custom file paths */[/COLOR][CR][ [COLOR red]\"My own one\"[/COLOR], [COLOR red]\"/path/to/glsl/my.frag.glsl\"[/COLOR], [COLOR red]\"audio\"[/COLOR], [COLOR red]\"/path/to/pngs/my_image.png\"[/COLOR], [COLOR red]\"\"[/COLOR], [COLOR red]\"\"[/COLOR] ],[CR][COLOR gray]/* Preset only available to OpenGL (see 7)  */[/COLOR][CR][ [COLOR red]\"Polar Beats by sauj123\"[/COLOR], [COLOR red]\"polarbeats.frag.glsl\"[/COLOR], [COLOR red]\"audio\"[/COLOR], [COLOR red]\"\"[/COLOR], [COLOR red]\"\"[/COLOR], [COLOR red]\"\"[/COLOR], [COLOR red]\"gl_only\"[/COLOR] ],[CR][COLOR gray]/* GLSL and PNG files are in the same folder as presets.json */[/COLOR][CR][ [COLOR red]\"Another one\"[/COLOR], [COLOR red]\"./another.frag.glsl\"[/COLOR], [COLOR red]\"./my_01.png\"[/COLOR], [COLOR red]\"./my_02.png\"[/COLOR], [COLOR red]\"audio\"[/COLOR], [COLOR red]\"\"[/COLOR] ],[CR][COLOR gray]/* Select texture image by file name, use default \"resources/\" dir */[/COLOR][CR][ [COLOR red]\"Fractal Land by Kali\"[/COLOR], [COLOR red]\"fractalland.frag.glsl\"[/COLOR], [COLOR red]\"tex02.png\"[/COLOR], [COLOR red]\"tex15.png\"[/COLOR], [COLOR red]\"audio\"[/COLOR], [COLOR red]\"\"[/COLOR] ][CR]][COLOR green]}[/COLOR][/COLOR]"
```

Comment:
Is such a long string with so many tags possible to translate?

Reply:
Not sure, thing by this maybe some says no, as really hard to understand what's there inside.
Only not found a way to make easy readable by user and also nice shown on text window on Kodi. Specially this color set on code example, makes it much more easy and understandable for Users on view by Window.

@gade01 how does it works with strings.po and added linebreaks in scripts and also on Weblate?

Can on Kodi e.g. to use as:
```po
msgctxt "#30038"
msgid "[B][I]JSON file for user defined shader lists[/I][/B]\n"
"\n"
"The file [B]presets.json[/B] is used to define custom shaders.\n"
"A custom path to the shader files (GLSL) can be declared in the add-on settings.[CR][CR]Contents of [B]presets.json[/B] should be:\n"
"[I]  1.[/I] Name of preset. Either hardcoded string or defined in [I]resources/language/.../strings.po[/I]\n"
"[I]  2.[/I] GLSL shader to use. if no custom path was declared, the default [B]resources/[/B] will be used. Set custom path to './' to use shaders from the same path as your [B]presets.json[/B] file.\n"
"[I]  3.[/I] Channel value 0. Defines the texture to use (e.g. PNG file). Can also be set to \"audio\" to pass related stream data within texture. The PNG and GLSL files should be located in the same directory.\n"
"[I]  4.[/I] Channel value 1 definition.\n"
"[I]  5.[/I] Channel value 2 definition.\n"
"[I]  6.[/I] Channel value 3 definition.\n"
"[I]  7.[/I] [I][Optional][/I] append string [B]\"gl_only\"[/B] to make this preset only available to OpenGL.\n"
"\n"
"[B]Example:[/B]\n"
"[COLOR white][COLOR green]{[/COLOR] [COLOR blue]\"presets\"[/COLOR]:[\n"
"[COLOR gray]/* Default way used by the addon itself */[/COLOR]\n"
"[ [COLOR red]\"The Disco Tunnel by poljere\"[/COLOR],  [COLOR red]\"discotunnel.frag.glsl\"[/COLOR], [COLOR red]\"tex02.png\"[/COLOR], [COLOR red]\"tex15.png\"[/COLOR], [COLOR red]\"audio\"[/COLOR], [COLOR red]\"\"[/COLOR] ],\n"
"[COLOR gray]/* Description defined in strings.po */[/COLOR]\n"
"[ 30100, [COLOR red]\"audioeclipse.frag.glsl\"[/COLOR], [COLOR red]\"audio\"[/COLOR], [COLOR red]\"\"[/COLOR], [COLOR red]\"\"[/COLOR], [COLOR red]\"\"[/COLOR] ],\n"
"[COLOR gray]/* Define custom file paths */[/COLOR]\n"
"[ [COLOR red]\"My own one\"[/COLOR], [COLOR red]\"/path/to/glsl/my.frag.glsl\"[/COLOR], [COLOR red]\"audio\"[/COLOR], [COLOR red]\"/path/to/pngs/my_image.png\"[/COLOR], [COLOR red]\"\"[/COLOR], [COLOR red]\"\"[/COLOR] ],\n"
"[COLOR gray]/* Preset only available to OpenGL (see 7)  */[/COLOR]\n"
"[ [COLOR red]\"Polar Beats by sauj123\"[/COLOR], [COLOR red]\"polarbeats.frag.glsl\"[/COLOR], [COLOR red]\"audio\"[/COLOR], [COLOR red]\"\"[/COLOR], [COLOR red]\"\"[/COLOR], [COLOR red]\"\"[/COLOR], [COLOR red]\"gl_only\"[/COLOR] ],\n"
"[COLOR gray]/* GLSL and PNG files are in the same folder as presets.json */[/COLOR]\n"
"[ [COLOR red]\"Another one\"[/COLOR], [COLOR red]\"./another.frag.glsl\"[/COLOR], [COLOR red]\"./my_01.png\"[/COLOR], [COLOR red]\"./my_02.png\"[/COLOR], [COLOR red]\"audio\"[/COLOR], [COLOR red]\"\"[/COLOR] ],\n"
"[COLOR gray]/* Select texture image by file name, use default \"resources/\" dir */[/COLOR]\n"
"[ [COLOR red]\"Fractal Land by Kali\"[/COLOR], [COLOR red]\"fractalland.frag.glsl\"[/COLOR], [COLOR red]\"tex02.png\"[/COLOR], [COLOR red]\"tex15.png\"[/COLOR], [COLOR red]\"audio\"[/COLOR], [COLOR red]\"\"[/COLOR] ]\n"
"][COLOR green]}[/COLOR][/COLOR]"
msgstr ""
```

But seen that you changed everywhere to a single line and I thought other ways not possible.

Also a part where I hope to become "Markdown" support added to Kodi, for cases like there (where planned from me since years :cry: ).